File: benchmarks/c/mix.c
static int mix(int n) {
  int i = 0;
  int acc = 0;
  while (i < n) {
    acc = (acc + (i * 31)) % 1000003;
    i = i + 1;
  }
  return acc;
}

int main(void) {
  return mix(10000000);
}

File: benchmarks/rust/mix.rs
fn mix(n: i32) -> i32 {
    let mut i = 0;
    let mut acc = 0;
    while i < n {
        acc = (acc + (i * 31)) % 1_000_003;
        i += 1;
    }
    acc
}

fn main() {
    std::process::exit(mix(10_000_000));
}
